hypothesis-python/tests/cover/test_pretty.py

```diff
@@ -631,3 +623,17 @@ def test_empty_printer():
 
 def test_breakable_at_group_boundary():
     assert "\n" in pretty.pretty([[], "000000"], max_width=5)
+
+
+@pytest.mark.parametrize(
+    "obj, rep",
+    [
+        (float("nan"), "nan"),
+        (-float("nan"), "-nan"),
+        (SIGNALING_NAN, "nan  # Saw 1 signaling NaN"),
```

Comment:
It may be good to cover those very rare scenarios of organically generated signaling NaNs (requires `from hypothesis.internal.floats import int_to_float` too)
```suggestion
        (SIGNALING_NAN, "nan  # Saw 1 signaling NaN"),
        (int_to_float(0x7FF8_0000_0000_0042), "nan  # Saw 1 signaling NaN"),
```
(but yeah I gave up trying to reproduce organic snans locally)